### ISSUE TITLE
{Freeway-Bug}[SampleCard-WeatherCompact; Config-Windows-timeline: The color contrast ratio for the text "Redmond, WA" is 2.82:1 which is not sufficient.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8471]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1

Pre-requisite:
Import-Sample Cards->WeatherCompact.json; Config-Windows-timeline

Repro Steps:

1. Navigate to the above sample card.
2. Observe the contrast issue for the text "Redmond, WA"

Actual Result:
The color contrast ratio for the text "Redmond, WA" is 2.82:1 which is not sufficient.

Expected Result:
The color contrast ratio for the text "Redmond, WA" should be at least 4.5:1

User Experience:
Low vision users will get impact and will not able to tell the text if its color contrast ratio is not sufficient.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![MAS1 4 3The color contrast ratio for the text Redmond, WA is 2 821 which is not sufficient](https://user-images.githubusercontent.com/89897257/232737959-e6dd2770-eb32-4988-9e9a-755abff25762.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
duplicate issue #8473

**Comment by vagpt** [comment]
Adding A11yTTValidated as per the above comment.
